src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -324,9 +352,10 @@ class ContentSidebar extends PureComponent<Props, State> {
             <Internationalize language={language} messages={messages}>
                 <div id={this.id} className={`be bcs ${className}`}>
                     <div className='be-app-element'>
-                        {file ? (
+                        {file && versions ? (
```

Comment:
Check should ideally not needed here, sidebar should render before versions is got